packages/react/src/authentication/contexts/UserProfileProvider.js

```diff
@@ -62,8 +65,8 @@ const reducer = (state, action) => {
  *
  * @example <caption>Adding the UserProfileProvider to your app</caption>
  *
- * import { AuthenticationProvider, UserProfileProvider } from '@farfetch/blackout-react/authentication/contexts';
- * import { useUserProfile } from '@farfetch/blackout-react/authentication/hooks';
+ * import { AuthenticationProvider, UserProfileProvider } from '@fps/react/authentication/contexts';
```

Comment:
Careful here! Make sure you don't have `@fps` anywhere else 🚨

Reply:
Yep removed it, also tried to find more references but couldn't find them